spec/controllers/users_controller_spec.rb

```diff
@@ -48,28 +62,31 @@
       it 'sets flash[:notice]' do
         expect(flash[:notice]).to be_a String
       end
-    end
 
-    context 'with invalid input' do
-      before { post :create, user: {email: '', full_name: '', password:''} }
-
-      it 'does not save @user to the database' do
-        expect(User.find_by email: user_params[:email]).to eq nil
+      it 'sends an email with valid input' do
+        expect(ActionMailer::Base.deliveries).to_not be_empty
```

Comment:
The inverse of `to_not be_empty` is `to be_present`. It is always better to use a positive matcher rather than a double negative.